hw/top_darjeeling/sw/autogen/chip/top_darjeeling.rs

```diff
@@ -1734,94 +1734,96 @@ pub const PLIC_INTERRUPT_FOR_PERIPHERAL: [PlicPeripheral; 159] = [
 #[derive(Copy, Clone, PartialEq, Eq)]
 #[repr(u32)]
 pub enum AlertPeripheral {
+    /// Unknown Peripheral
+    Unknown = 0,
```

Comment:
How about _External_, since this is all we know about the source of these alerts?

Reply:
should that be done for interrupts as well?

Reply:
@andreaskurth I changed it to Incoming.

Reply:
I'm not convinced that _incoming_ is more descriptive than _external_. Aren't all alerts _incoming_ from the perspective of alert handling HW and SW?

Reply:
Not at all! When designing at the top level, I think incoming and outgoing make full sense. The code I commented on enumerates PLIC interrupts, though, and I think there _external_ is more descriptive than _incoming_, because for the PLIC every alert is incoming, but only alerts that are incoming at the top level design are external.

Does that make sense, or am I overthinking this?